src/targets/gpu/kernels/include/migraphx/kernels/float8.hpp

```diff
@@ -71,7 +71,7 @@ struct float8
 
     __device__ explicit constexpr float8(uint8_t bits, from_bits_t) : data(bits) {}
 
-#if defined(__gfx940__) || defined(__gfx941__) || defined(__gfx942__)
+#if defined(__gfx942__)
```

Comment:
I believe this file is actually taken from another rocm source. Should be fine to change this but should remove this code and use the new hip type. https://github.com/ROCm/AMDMIGraphX/issues/3890